tests/animal/schema.sql

```diff
@@ -0,0 +1,7 @@
+create table animal (
+  id serial primary key
+);
+
+create table dog (
+  id integer primary key references animal
+);
```

Comment:
Lets add a few more examples to see that sensible behaviour continues even if there are many relationships.

```suggestion
create table cat (
  id integer primary key references animal
);

create table dog (
  id integer primary key references animal
);

create table gerbil (
  animal_id integer primary key references animal
);
```

Reply:
Added.